syntax = "proto3";

package protobuff;

message EmailIdPb {
    string localPart = 1;
    string domain = 2;
}

enum EmailTypeEnum {
    UNKNOWN_EMAIL = 0;
    WELCOME_MAIL = 1;
    VERIFICATION_MAIL = 2;
}

message EmailBuilderPb {
    EmailTypeEnum emailType = 1;
    EmailIdPb fromId = 2;
    repeated EmailIdPb toId = 3;
    string subject = 4;
    string content = 5;
    string from = 6;
    string to = 7;
    string name = 8;
}
